homeassistant/components/incomfort/water_heater.py

```diff
@@ -0,0 +1,96 @@
+"""Support for an Intergas boiler via an InComfort/Intouch Lan2RF gateway."""
+import asyncio
+import logging
+
+from homeassistant.components.water_heater import WaterHeaterDevice
+from homeassistant.const import TEMP_CELSIUS
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+INCOMFORT_SUPPORT_FLAGS = 0
+
+INCOMFORT_MAX_TEMP = 80.0
+INCOMFORT_MIN_TEMP = 30.0
+
+
+async def async_setup_platform(hass, hass_config, async_add_entities,
+                               discovery_info=None):
+    """Set up an InComfort/Intouch water_heater device."""
+    client = hass.data[DOMAIN]['client']
+    heater = hass.data[DOMAIN]['heater']
+
+    async_add_entities([
+        IncomfortWaterHeater(client, heater)], update_before_add=True)
+
+
+class IncomfortWaterHeater(WaterHeaterDevice):
+    """Representation of an InComfort/Intouch water_heater device."""
+
+    def __init__(self, client, boiler):
+        """Initialize the water_heater device."""
+        self._client = client
+        self._boiler = boiler
+        self._name = 'Boiler'
```

Comment:
Boiler can be extracted to a module level constant.

Reply:
```python
self._name = BOILER_NAME
```

Reply:
Nah, just return `BOILER_NAME` directly in the name property.

Reply:
Sorry, I can see now how it's stupid idea - I think this is one of those things that will make more sense when the other components/platforms are added to the integration.

Anyway, will do as you suggest.